0.15/alpine/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN adduser -S bitcoin
 ENV BERKELEYDB_VERSION=db-4.8.30.NC
 ENV BERKELEYDB_PREFIX=/opt/${BERKELEYDB_VERSION}
 
-ENV BITCOIN_VERSION=0.15.0
+ENV BITCOIN_VERSION=0.15.0.1
```

Comment:
Please add the new 0.15.* versions to README.

Reply:
Fixed, thanks.